root/user/UserProfile.js

```diff
@@ -855,6 +858,16 @@ const UserProfile = ({
   const viewingOwnProfile = $c.user != null && $c.user.id === user.id;
   const adminViewing = $c.user != null && isAccountAdmin($c.user);
   const encodedName = encodeURIComponent(user.name);
+  const restrictions = [];
+  if (isEditingDisabled(user)) {
+    restrictions.push(l('Editing/voting disabled'));
+  }
+  if (isAddingNotesDisabled(user)) {
+    restrictions.push(l('Edit notes disabled'));
+  }
+  if (isUntrusted(user)) {
+    restrictions.push(l('Untrusted'));
+  }
```

Comment:
I'd check `adminViewing` before pushing anything onto `restrictions` at all, since it isn't used otherwise and ensures we don't accidentally show those values anywhere we're not supposed to.
```suggestion
  if (adminViewing) {
    if (isEditingDisabled(user)) {
      restrictions.push(l('Editing/voting disabled'));
    }
    if (isAddingNotesDisabled(user)) {
      restrictions.push(l('Edit notes disabled'));
    }
    if (isUntrusted(user)) {
      restrictions.push(l('Untrusted'));
    }
  }
```

Reply:
Commited the suggestion, we can squash on merge if it seems otherwise fine.